WebDriverAgentLib/Categories/NSExpression+FBFormat.m

```diff
@@ -17,14 +17,14 @@ + (instancetype)fb_wdExpressionWithExpression:(NSExpression *)input
   if ([input expressionType] != NSKeyPathExpressionType) {
     return input;
   }
-  NSString *actualPropName = [input keyPath];
-  NSUInteger dotPos = [actualPropName rangeOfString:@"."].location;
+  NSString *propName = [input keyPath];
+  NSUInteger dotPos = [propName rangeOfString:@"."].location;
   if (NSNotFound != dotPos) {
-    actualPropName = [actualPropName substringToIndex:dotPos];
-    NSString *suffix = [actualPropName substringFromIndex:dotPos];
+    NSString *actualPropName = [propName substringToIndex:dotPos];
+    NSString *suffix = [propName substringFromIndex:(dotPos + 1)];
     return [NSExpression expressionForKeyPath:[NSString stringWithFormat:@"%@.%@", [FBElementUtils wdAttributeNameForAttributeName:actualPropName], suffix]];
```

Comment:
could you please add unit test for such case? The tests container is located in UnitTests folder, NSExpressionFBFormatTests.m file.